Documentation/contributing/release/feature.rst

```diff
@@ -136,3 +136,7 @@ For the final release
 
 #. Update ``SECURITY.md`` to represent the security support for the most recent
    three release series.
+
+#. Set the branch for the oldest release as "Active" and "Hidden" in the
+   readthedocs Admin page. (Replace ``v1.2`` with the right branch)
+   ``https://readthedocs.org/dashboard/cilium/version/v1.2/``
```

Comment:
@aanm it seems like we actually have just unmarked "Active" which deletes the old docs for older branches. I'm not sure if that was intentional. Should we do that instead?

It's the difference between whether the branch's docs remain accessible by link but hidden from the main docs pages (and maybe from search engines...?) vs. Not available at all.

Reply:
@joestringer I think it would make sense to mark them as non "HIdden". Sometimes it is useful for users to still have a way to find those docs without typing them in the URL bar IMO.

Reply:
@aanm How do we avoid that list getting massive though?

One thought in the back of my mind here is that having a limited set of versions linked in the bottom right gives a subtle clue to the reader that these are the supported versions, and if you're looking for documentation relating to older releases then you're on your own for support.

I don't think the RTD plugin for that box in the bottom right supports a notion of listing versions by EOL.

If I compare to something like python docs, they only list the supported + last EOL (python2.7) from their main page, then they have a separate page that lists all old docs versions/links.

Reply:
Let's compromise and have all `>=vX.Y-1` non-hidden where `vX.Y` is the oldest supported version, in this case, which today is 1.6, so we should keep the docs available with up to the version 1.5. WDYT?

Reply:
Seems reasonable, but what's special about vX.Y-1? Each version we expose will have upgrade instructions for the prior two releases. Are there other aspects of the documentation that you're wanting to ensure are still available from the latest EOL release (eg from v1.5 docs right now)?

The proposal would mean the following versions would have docs links:
* v1.8
* v1.7
* v1.6
* v1.5 (EOL, but we don't have a way to specifically highlight this in the documentation right now)

Via v1.5, there would be upgrade instructions from:
* v1.4
* v1.3

There's also the aspect that we've only backported the docs theme to v1.6 so v1.5 will appear differently which is a bit weird from a UI perspective.

Reply:
> Are there other aspects of the documentation that you're wanting to ensure are still available from the latest EOL release (eg from v1.5 docs right now)?

That's the thing, I don't know. If for example we remove the docs for 1.6 once we release Cilium 1.9, users won't be able to get the documentation for 1.6 for example.

Reply:
Then maybe we should introduce a dedicated page with links for all of the old docs.

Reply:
> If for example we remove the docs for 1.6 once we release Cilium 1.9, users won't be able to get the documentation for 1.6 for example.

That's kinda the point though right? When Cilium 1.9 comes out, Cilium 1.6 is no longer supported. If someone figures out they can change the URL to get v1.6 docs then good for them, otherwise they should upgrade or build the docs themselves..?

> Then maybe we should introduce a dedicated page with links for all of the old docs.

I briefly looked into this, and I don't see a great option to do this. I don't want to expand the navbar on left any bigger as the entire thing fits within a standard 1080p screen; but then if we start nesting the links under somewhere, it becomes more difficult to find which makes it more pointless.